.github/workflows/build.yml

```
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [0.12.x, 4.x, 6.x, 8.x, 12.x, 13.x, 14.x, 15.x]
+        node-version: [12.x, 13.x, 14.x, 15.x]
```

```suggestion
        node-version: [12.x, 13.x, 14.x, 15.x, 16.x]
```

Might want to start testing on Node 16 as well?

---

Yes, we should start checking for Node.js v16.x too.